.ci/install_cloud_hypervisor.sh

```diff
@@ -15,63 +15,15 @@ readonly script_dir=$(dirname $(readlink -f "$0"))
 cidir=$(dirname "$0")
 arch=$("${cidir}"/kata-arch.sh -d)
 source "${cidir}/lib.sh"
-# Where real kata build script exist, via docker build to avoid install all deps
-latest_build_url="${jenkins_url}/job/cloud-hypervisor-nightly-$(uname -m)/${cached_artifacts_path}"
-clh_bin_name="cloud-hypervisor"
-clh_install_path="/usr/bin/${clh_bin_name}"
-cloud_hypervisor_repo=$(get_version "assets.hypervisor.cloud_hypervisor.url")
-go_cloud_hypervisor_repo=${cloud_hypervisor_repo/https:\/\//}
 
-install_clh() {
-	[ -n "$cloud_hypervisor_repo" ] || die "failed to get cloud_hypervisor repo"
-	export cloud_hypervisor_repo
-
-	# Get version for cloud_hypervisor from runtime/versions.yaml
-	cloud_hypervisor_version=$(get_version "assets.hypervisor.cloud_hypervisor.version")
-	[ -n "$cloud_hypervisor_version" ] || die "failed to get cloud_hypervisor version"
-	export cloud_hypervisor_version
-
-	# Get cloud_hypervisor repo
-	go get -d "${go_cloud_hypervisor_repo}" || true
-	# This may be downloaded before if there was a depends-on in PR, but 'go get' wont make any problem here
+main() {
+	# Just in case the kata-containers repo is not cloned yet.
 	clone_katacontainers_repo
-	pushd  $(dirname "${GOPATH}/src/${go_cloud_hypervisor_repo}")
-	# packaging build script expects run in the hypervisor repo parent directory
-	# It will find the hypervisor repo and checkout to the version exported above
-	"${katacontainers_repo_dir}/tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh"
-	sudo install -D "cloud-hypervisor/${clh_bin_name}"  "${clh_install_path}"
-	popd
-}
-
-install_prebuilt_clh() {
-	local checksum_file="sha256sum-cloud-hypervisor"
-	go get -d "${go_cloud_hypervisor_repo}" || true
-	pushd  $(dirname "${GOPATH}/src/${go_cloud_hypervisor_repo}")
 
-	curl -fsOL --progress-bar "${latest_build_url}/${clh_bin_name}" || return 1
-	curl -fsOL "${latest_build_url}/${checksum_file}" || return 1
-
-	info "Verify download checksum"
-	sudo sha256sum -c "${checksum_file}" || return 1
-
-	info "installing ${clh_bin_name}" "${clh_install_path}"
-	sudo install -D ${clh_bin_name} "${clh_install_path}"
+	pushd $katacontainers_repo_dir
+	sudo -E PATH=$PATH make cloud-hypervisor-tarball
```

Comment:
- Shouldn't this be something like:
  ```bash
  make -C tools/packaging/kata-deploy/local-build -n cloud-hypervisor-tarball
  ```
- Do we actually need `sudo` to build the archive file?

Reply:
Ack - thanks for the explanation @fidencio !